test/copy.bats

```diff
@@ -0,0 +1,40 @@
+load helpers
+
+function setup() {
+    stacker_setup
+}
+
+function teardown() {
+    cleanup
+}
+
+@test "built type layers are restored correctly" {
+    cat > stacker.yaml <<EOF
+parent:
+    from:
+        type: oci
+        url: $CENTOS_OCI
+    run: |
+        touch /root/parent
+        cat /proc/self/mountinfo
+child:
+    from:
+        type: built
+        tag: parent
+    copy:
+      - /stacker/root/parent -> /root/parent
```

Comment:
I don't follow this - how did /stacker/root/parent get there? it wasn't imported.
/root/parent should already be there when the child container is started because it is  in the parent rfs, right?

IIUC the use case for copy is to copy something from outside the container in, as in dockerfile's `COPY`.
For that use case we could also consider e.g. adding a `destination` field to `import` directives:

```yaml
import:
  - path: foo.conf
    hash: <optional>
    dest: /etc/foo/foo.conf
```